gene_finder.py

```diff
@@ -30,9 +29,15 @@ def get_complement(nucleotide):
     >>> get_complement('C')
     'G'
     """
-    # TODO: implement this
-    pass
-
+    nucleotide_inputs = ['A', 'T', 'C', 'G']
+    nucleotide_complements = ['T', 'A', 'G', 'C']
+    i = 0
+    complement = 'x' #lets the user know the complement was incorrectly computed
+    while i < len(nucleotide_inputs):
+        if nucleotide_inputs[i] == nucleotide:
+            complement = nucleotide_complements[i]
+        i += 1
+    return complement
```

Comment:
I like how you implemented this function without using any `if` or `else if` statements. Now that you've learned what dictionaries are, you could do something like this as well : 
`nucleotide_complements' = {'A':'T', 'T':'C', 'C':'G', 'G':'C'}`
` return nucleotide_complements[nucleotide]`